pkg/lib/cockpit-components-privileged.jsx

```diff
@@ -34,7 +34,7 @@ import cockpit from "cockpit";
  */
 export function Privileged({ excuse, allowed, placement, tooltipId, children }) {
     // wrap into extra <span> so that a disabled child keeps the tooltip working
-    let contents = <span id={tooltipId}>{ children }</span>;
+    let contents = <span id={tooltipId} data-test-has-tooltip={allowed ? null : ""}>{ children }</span>;
```

Comment:
I think the `div.tooltip` only appears once the tooltip is actually open.  We also need to wait until the `Overlaytrigger` has been put into the DOM so that we can send the `mouseover` that will open the tooltip.

Reply:
We don't do weird things in the other tests that check tooltips. `b.mouse()` already waits for the element to become present and visible.

Reply:
In the other cases, the test is in a state where we know that the tooltip is in the DOM.  For example, after being joined to a domain, the hostname edit button is expected to have a tooltip. We don't need to wait for it.

In the TestInstall cases, there is no explicit action that the tests do that would guarantee that the tooltip is there.  It really appears asynchronously and we need to wait for it. The old code also did this, with             
```
b.wait_present("#system-info-domain [data-original-title]")
```

Reply:
Ah, you mean this doesn't wait for the tooltip, but it waits for the page to get into a state where the action of hovering *will cause* the tooltip to appear. That's the bit that I was missing, sorry.